conformance/tests/annotations_coroutines.py

```diff
@@ -16,7 +16,10 @@ async def func1(ignored: int, /) -> str:
     return "spam"
 
 
-assert_type(func1, Callable[[int], Coroutine[Any, Any, str]])
+# Don't use assert_type here because some type checkers infer
+# the narrower type types.CoroutineType rather than typing.Coroutine
```

Comment:
This can wait but I'd favor making the conformance tests test for CoroutineType here in the future. It's the precise type at runtime, and ideally it's what type checkers should infer.